socorro/external/crashstorage_base.py

```diff
@@ -652,6 +652,120 @@ def new_crashes(self):
 
 
 #==============================================================================
+class MigrationCrashStorage(FallbackCrashStorage):
+    required_config = Namespace()
+    required_config.add_option(
+        'date_threshold',
+        default="150401",
```

Comment:
This is great!  Would it be prudent to check that this string conforms to our expectations about date format? I'd hate for someone to fat finger the config.